spec/e2e/has_one_spec.rb

```diff
@@ -28,6 +28,20 @@
       it 'raises an error when trying to create a relationship' do
         expect { unsaved_node.parent = HasOneA.create }.to raise_error(Neo4j::ActiveNode::HasN::NonPersistedNodeError)
       end
+
+      context 'with enabled auto-saving' do
+        before  { Neo4j::Config[:autosave_on_assignment] = true }
+        after   { Neo4j::Config[:autosave_on_assignment] = false }
```

Comment:
This should probably be set to whatever `Neo4j::Config[:autosave_on_assignment]` was before it was set to `true`.  We should also probably have a spec helper for this, though you don't need to do that if you don't want to ;)